### PR TITLE
TECH-891 add confirmation for bulk status changes

### PR DESCRIPTION
**Pull request by ericbroder**

### CODE REVIEW

File: nominations/admin.py [hunk 1]
```diff
@@ -1,12 +1,18 @@
+from django.conf.urls import include, url
 from collections import OrderedDict
 from django.conf import settings
 from django.contrib import admin
+from django.urls import reverse_lazy
```
Comment (cjmabry) [inline nominations/admin.py]: Not used, remove.

File: nominations/admin.py [hunk 2]
```diff
@@ -1,12 +1,18 @@
+from django.conf.urls import include, url
```
Comment (cjmabry) [inline nominations/admin.py]: `include` not used, remove.

File: nominations/admin.py [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
 from django.contrib import admin
+from django.urls import reverse_lazy
 from datetime import datetime
 from django.db.models import Q
 from .models import *
+from .admin_views import ApplicationPDFView, ApplicationsStatusChangeView
 from local_groups.models import Group
 from local_groups.actions import export_as_csv_action
 import pprint
+from django.contrib.contenttypes.models import ContentType
```
Comment (cjmabry) [inline nominations/admin.py]: Not used, remove.

File: nominations/admin.py (hunk 2 above)
Comment (cjmabry) [inline nominations/admin.py]: `include` not used, remove.

File: nominations/admin.py (hunk 1 above)
Comment (cjmabry) [inline nominations/admin.py]: Not used, remove.

File: nominations/models.py [hunk 4]
```diff
@@ -617,6 +617,12 @@ def save(self, *args, **kwargs):
         super(Application, self).save(*args, **kwargs)
 
     class Meta:
+        permissions = (
```
Comment (cjmabry) [inline nominations/models.py]: Nice.

Comment (ericbroder) [reply nominations/models.py]: Yeah I think we should do more of this. We have lots of hardcoded access control and some of it should probably use permissions instead.

**Comment by ericbroder** [comment]
@cjmabry I suggest in case like this better to approve PR but don't merge, and then I can make minor changes and merge. "Request change" function I think is better for critical issues, not minor stuff like unused packages.

**Pull request merged**